app/src/main/java/com/example/drawingApp/MainActivity.kt

```diff
@@ -10,14 +16,40 @@ class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
-
         val colorCircleView = findViewById<SelectedColorView>(R.id.colorCircle)
         val drawingFieldView = findViewById<DrawingFieldView>(R.id.drawField)
+        findViewById<TextView>(R.id.newLayer).setOnClickListener {
+            layerAlertDialog()
+        }
 
         colorCircleView.onColorChange = { stateColor ->
             model.primaryColor = stateColor
             drawingFieldView.updateColor(stateColor)
         }
 
     }
+
+    fun layerAlertDialog() {
+        val inflater = LayoutInflater.from(this@MainActivity)
+        val view = inflater.inflate(R.layout.layer_dialog, null)
+        val layerField = view.findViewById<EditText>(R.id.newLayerField)
+        layerField.setText(getString(R.string.new_layer_hint, model.layers.size + 1))
```

Comment:
This doesn't fit the ticket requirement to only increment on layers stored of the type "Layer X" does it?

Reply:
fixed, it should now correctly update on the correct layer number